hr_timesheet_hours_billed/models/hours_billed.py

```diff
@@ -0,0 +1,27 @@
+from odoo import api, fields, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    approved = fields.Selection(
+        selection=[("no", "No"), ("yes", "Yes")],
+        default="no",
+        required=True,
+    )
+    approved_uid = fields.Many2one(
+        "res.users", index=True, tracking=True, default=lambda self: self.env.user
```

Comment:
This will record approval user only for the first time.
We need to track also in case if approval is removed and then added again.